### ISSUE TITLE
(</>) UI mistakes and lack of information.

### ISSUE THREAD

**Issue by PiyushK79** [Amyth0007/ReactRide#9]
So many UI issues and typing mistakes, Not much information about what the application does. User does not understand at the start that for what use is the application. Assign his issue to me under HacktoberFest 2023 so I can start working on it.

**Comment by Amyth0007** [comment]
Yeah sure, you can put one by one issue here and start working on it 👍

**Comment by PiyushK79** [comment]
Alright! Can u please put the related tags i.e for hactoberfest2023 to it? Thanks.

**Comment by Sujit-magar** [comment]
Let's work together to enhance the UI and ensure that all necessary information is readily available. @Amyth0007 @PiyushK79

**Comment by Amyth0007** [comment]
okay you can work together !!

**Comment by PiyushK79** [comment]
This repo has been excluded from Hacktoberfest. @Amyth0007 have you looked into this issue?
